imports/ui/components/CardList.jsx

```diff
@@ -13,6 +13,48 @@ class CardList extends Component {
     this.props.getItems({});
   }
 
+  sortByPricePressed = () => {
+    var items = this.props.items.items;
+
+    if (items.length > 1) {
+      items.sort(function(a, b) {
+        // TODO: Refactor after card with $h is remove from database
+        let aValue =
+          a.price.substr(1) && parseFloat(a.price.substr(1))
+            ? parseFloat(a.price.substr(1))
+            : 0;
+        let bValue =
+          b.price.substr(1) && parseFloat(b.price.substr(1))
```

Comment:
Just wondering if this is because price is saved as "$ {number}" right now? Should we make the price to be just a number?

Reply:
I i agree! I wonder if we should implement the unit selection first?

Reply:
@YaoLiu93 I've considered the db sort priorities but went with front end sorting because then call to the backend is made once is less expensive than doing a bunch of DB network calls. But our project is easy enough to do either, JS does additional sort priorities as you can see in my code, we sort by price then rating. 

@YaoLiu93 @inquisitiv-e Yes this is because it is a "$ {number}" string right now from Yao's implementation. Either stored as "$ {number}" string or change value to be stored as number and later in jsx add $ back are both fine with me.

Reply:
@YaoLiu93 @inquisitiv-e Sure I can add in the unit selection. We should talk about the design, are units going to be a dropdown? how are we going to compare an item with one unit with another item with a different unit?

Reply:
Carry out discussion about unit selection in this task since this PR is for sorting: https://github.com/FreshPrice/Fresh/issues/33

Reply:
> Just wondering if this is because price is saved as a string, "$ {number}" right now? Should we make the price to be just a number?

Refactored code, price is number now